scripts/tiqit/defaults.py

```diff
@@ -22,7 +22,7 @@ def _encodeValue(field, data):
     return '&'.join(map(encodeHTML, vals))
 
 def _decodeValue(field, value):
-    return dict(zip(field.defaultsWith, map(decodeHTML, value.split('&'))))
+    return dict(list(zip(field.defaultsWith, list(map(decodeHTML, value.split('&'))))))
```

Comment:
Ergh, another `dict(list(...))` thing that should probably be a comprehension of some form